academy/1-what-is-cosmos/atom-staking.md

```diff
@@ -46,7 +46,7 @@ When Keplr is installed it should be available in the list of extensions when yo
 <HighlightBox type="warn">
 
 When you own ATOM with Keplr, **you are in charge**. There is no one you can call if _you forget your password/seed phrase or you send tokens to the wrong address_.
-
+<br></br>
```

Comment:
Can something equivalent not be done during the compilation? @coldice

Reply:
It is a bit fighting against the Markdown spec - Markdown doesn't allow multiple blank lines. For compatibility, it is better to use these specific html tags instead. We were thinking about automatically converting multiple empty lines into br, but it is context-dependent.